long YAML parse error

```
Traceback (most recent call last):
  File "./createcomments.py", line 14, in <module>
    definitions = yaml.safe_load(sys.stdin)
  File "/usr/lib/python2.7/dist-packages/yaml/__init__.py", line 93, in safe_load
    return load(stream, SafeLoader)
  File "/usr/lib/python2.7/dist-packages/yaml/__init__.py", line 71, in load
    return loader.get_single_data()
  File "/usr/lib/python2.7/dist-packages/yaml/constructor.py", line 37, in get_single_data
    node = self.get_single_node()
  File "/usr/lib/python2.7/dist-packages/yaml/composer.py", line 36, in get_single_node
    document = self.compose_document()
  File "/usr/lib/python2.7/dist-packages/yaml/composer.py", line 55, in compose_document
    node = self.compose_node(None, None)
  File "/usr/lib/python2.7/dist-packages/yaml/composer.py", line 82, in compose_node
    node = self.compose_sequence_node(anchor)
  File "/usr/lib/python2.7/dist-packages/yaml/composer.py", line 111, in compose_sequence_node
    node.value.append(self.compose_node(node, index))
  File "/usr/lib/python2.7/dist-packages/yaml/composer.py", line 84, in compose_node
    node = self.compose_mapping_node(anchor)
  File "/usr/lib/python2.7/dist-packages/yaml/composer.py", line 133, in compose_mapping_node
    item_value = self.compose_node(node, item_key)
  File "/usr/lib/python2.7/dist-packages/yaml/composer.py", line 82, in compose_node
    node = self.compose_sequence_node(anchor)
  File "/usr/lib/python2.7/dist-packages/yaml/composer.py", line 111, in compose_sequence_node
    node.value.append(self.compose_node(node, index))
  File "/usr/lib/python2.7/dist-packages/yaml/composer.py", line 84, in compose_node
    node = self.compose_mapping_node(anchor)
  File "/usr/lib/python2.7/dist-packages/yaml/composer.py", line 133, in compose_mapping_node
    item_value = self.compose_node(node, item_key)
  File "/usr/lib/python2.7/dist-packages/yaml/composer.py", line 69, in compose_node
    % anchor.encode('utf-8'), event.start_mark)
yaml.composer.ComposerError: found undefined alias 'name'
  in "<stdin>", line 16, column 9
Makefile:7: recipe for target 'sql/post/comments.sql' failed
make: *** [sql/post/comments.sql] Error 1
```

--- a/yaml2json.py
+++ b/yaml2json.py
@@ -2,6 +2,6 @@
 import sys, yaml, json
 try:
     json.dump(yaml.safe_load(sys.stdin), sys.stdout, indent=2, separators=(',', ': '))
-except yaml.YAMLError as e:
+except (yaml.YAMLError, yaml.composer.ComposerError) as e:
     sys.stderr.write("YAML error:\n%s\n" % e)
     sys.exit(1)

